packages/augur-core/source/contracts/WarpSync.sol

```diff
@@ -21,7 +21,7 @@ contract WarpSync is IWarpSync, Initializable {
 
     uint256 private constant MARKET_LENGTH = 1 days;
     uint256 private constant MAX_NUM_TICKS = 2 ** 256 - 2;
-    int256[] private PRICES = [0, 1 ether];
+    int256[] private PRICES = [int256(uint256(1) << 255), int256(~(uint256(1) << 255))];
```

Comment:
Wat?  At the least, this magic horribleness is deserving of a comment...

Reply:
Consider: `[int256(2**255), int256(2**255-1)]`.  Still not great, but maybe better?

Reply:
Another option for ultimate clarity would be:
```
int256 private INT256_MIN = int256(2**255);
int256 private INT256_MAX = int256(2**255 - 1);
int256[] private PRICES = [INT256_MIN, INT256_MAX];
```